libcodechecker/analyze/pre_analysis_manager.py

```diff
@@ -49,7 +48,7 @@ def collect_statistics(action, source, config, environ, statistics_data):
     LOG.debug_analyzer(cmd)
 
     ret_code, analyzer_out, analyzer_err = \
-        analyzer_base.SourceAnalyzer.run_proc(cmdstr)
+        analyzer_base.SourceAnalyzer.run_proc(cmdstr, env=environ)
```

Comment:
Maybe I just missed it but where is `environ` set to some value? It was an argument before.

Reply:
It is still an argument. It was just unused before. `build_stat_coll_cmd` did not use it.